src/app/file-explorer.js

```diff
@@ -76,11 +76,14 @@ function fileExplorer (appAPI, files) {
     </span>
   `
 
-  appUI.register('currentFile', fileFocus)
+  appUI.register('fileChanged', (changedFiles) => {
+    if (changedFiles[0] == 'currentFile') {
+      fileFocus(files.get('currentfile'))
```

Comment:
Does this mean that `currentfile` is a file in the regular files API? Also why is one camelcase and the other not?